lib/tasks/grda_warehouse.rake

```diff
@@ -218,13 +218,14 @@ namespace :grda_warehouse do
 
   # rake grda_warehouse:anonymize_client_names['var/data/IL504']
   desc 'Anonymize all client names in Client.csv'
-  task :anonymize_client_names, [:path] => [:environment, 'log:info_to_stdout'] do |task, args|
+  task :anonymize_client_names, [:path] => [:environment, 'log:info_to_stdout'] do |_task, args|
     raise 'path is required' unless args.path.present?
+
     path = args.path
     file = File.join(path, 'Client.csv')
     CSV.open("#{file.gsub('.csv', '.anon.csv')}", 'wb') do |csv|
       CSV.foreach(file, headers: true) do |row|
-        csv << row.headers() if $. == 2
+        csv << row.headers if $. == 2
         row['FirstName'] = "First_#{row['PersonalID']}"
```

Comment:
@eanders do you mind if I update this task to use the Faker code instead? Makes more sense to run it on the CSV before importing it, but I want the realistic data.

Reply:
I think that's fine, we don't seem to use this anywhere currently.

Reply:
It might make sense to cleanup the SSN in this as well.  Seems odd we'd change the names but not the SSNs.

Reply:
I will move the ssn/dob cleanup code up here as well.